### PR TITLE
scenarios refactor before adding mine to orchard

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: zingo-testutils/src/lib.rs [hunk 1]
```diff
@@ -576,49 +585,12 @@ pub mod scenarios {
...
-            recipient,
-            txid,
-        )
+    pub async fn faucet_default() -> (RegtestManager, ChildProcessHandler, LightClient) {
+        let regtest_network = zingoconfig::RegtestNetwork::all_upgrades_active();
+        faucet(regtest_network).await
     }
 
-    pub async fn two_wallet_one_miner_fund(
+    pub async fn faucet_recipient(
```
Comment (zancas) [inline zingo-testutils/src/lib.rs:593]: Can we make this something like `faucet_recipient_network`?

File: zingo-testutils/src/lib.rs [hunk 2]
```diff
@@ -653,7 +625,121 @@ pub mod scenarios {
         )
     }
 
-    pub async fn two_wallet_one_miner_fund_transparent(
+    pub async fn faucet_recipient_default() -> (
```
Comment (zancas) [inline zingo-testutils/src/lib.rs:628]: And then change this to just be `faucet_recipient`..  that way when I am writing my test I am (by default) just writing `faucet_recipient`..  and only typing extra characters in special cases where I need to specify the network upgrades...

File: zingo-testutils/src/lib.rs (hunk 1 above)
Comment (Oscar-Pepper) [reply zingo-testutils/src/lib.rs:593]: as you will see in the next PR (mining to orchard) these scenarios will not be limited by parameters. regtestnetwork is just the first of many parameters we think would be useful for setting up scenarios. i think default calling a general version makes more sense than using the general name by default and adding for example `_custom` to the general version. also custom_clients would become custom_clients_custom hehe

File: zingo-testutils/src/lib.rs (hunk 2 above)
Comment (Oscar-Pepper) [reply zingo-testutils/src/lib.rs:628]: see my other comment

**Review by zancas — APPROVED** [review]
This is great!

**Pull request merged**